### PR TITLE
Add tests for CLI

### PR DESCRIPTION
**Pull request by nonemax**

### CODE REVIEW

File: dbaas-cli/integtests/k8s-psmdb.go [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+		return errors.Wrap(err, "unmarshal json out")
+	}
+	return nil
+}
+
+func (psmdb *KuberPSMDB) DescribeDB() error {
+	fmt.Println("Run describe-db " + psmdb.dbName)
+	o, err := runCmd(psmdb.cmd, psmdb.subCmd, "describe-db", psmdb.dbName)
+	if err != nil {
+		return errors.Wrap(err, "run describe-db for"+psmdb.dbName+" cmd")
```
Comment (dAdAbird) [inline dbaas-cli/integtests/k8s-psmdb.go]: use https://pkg.go.dev/github.com/pkg/errors?tab=doc#Wrapf

File: dbaas-cli/integtests/k8s-psmdb.go [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+	}
+	fmt.Println(o)
+	if !strings.Contains(o, "ready") && !strings.Contains(o, psmdb.dbName) {
+		return errors.New("db not start correctly. Output: " + o)
```
Comment (dAdAbird) [inline dbaas-cli/integtests/k8s-psmdb.go]: use https://pkg.go.dev/github.com/pkg/errors?tab=doc#Errorf

File: dbaas-cli/integtests/main.go [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+	}
+	fmt.Println("Tests done")
+}
+
+func getTestEngines() []TestEngine {
+	cmd := "../cmd/percona-dbaas"
```
Comment (dAdAbird) [inline dbaas-cli/integtests/main.go]: how to get "../cmd/percona-dbaas" binary?

**Review by dAdAbird — CHANGES_REQUESTED** [review]
I'd rather check cluster, not cli output

File: dbaas-cli/integtests/main.go [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+	"os/exec"
+)
+
+type TestEngine interface {
+	Run() error
+	CreateDB(rootPass string) (o string, err error)
+	ListDB() error
+	DescribeDB() error
+	ModifyDB() error
+	DeleteDB(bool) error
```
Comment (dAdAbird) [inline dbaas-cli/integtests/main.go]: Are you sure you need these methods in the interface since only Run() is used of the objects?

**Review by dAdAbird — CHANGES_REQUESTED** [review]

File: dbaas-cli/integtests/main.go (hunk 3 above)
Comment (nonemax) [reply dbaas-cli/integtests/main.go]: bash script added

**Review by dAdAbird — APPROVED** [review]

**Pull request merged**